projects/experimental/components/pdf-viewer/pdf-viewer.template.html

```diff
@@ -0,0 +1,54 @@
+<ng-container [ngTemplateOutlet]="isMobile ? mobile : desktop" />
+
+<ng-template #mobile>
+    <tui-app-bar>
+        <button
+            tuiButton
+            tuiSlot="left"
+            type="button"
+            (click)="context.$implicit.complete()"
+        >
+            {{ close() }}
+        </button>
+        <ng-container [ngTemplateOutlet]="content" />
```

Comment:
It's off center:
<img width="898" height="209" alt="image" src="https://github.com/user-attachments/assets/ecaaa95d-0b39-4640-9c28-b878deb55162" />